### ISSUE TITLE
Wildrydes 

### ISSUE THREAD

**Issue by robertjsandor** [aws-samples/aws-serverless-workshops#323]
Problem with Wildrydes App

Hi, not even sure I'm asking this in the right spot, but I've been using the Wildrydes app to teach an intro to cloud class. Not sure when, but it looks like the map background, which I believe is supplied by ArcGIS Online is now failing. Don't know if they've made that private. Here's the error I'm getting:

Please sign in to access the item on ArcGIS Online (0e02e6f86d02455091796eaae811d9b5)

It's asking for a Username/Password.

Has this changed?

Thanks a million for any help,
Bob

**Comment by electron-volt** [comment]
I just noticed the same issue with the map. I can confirm it worked without any issues in January of 2022.

**Comment by joohan-lee** [comment]
I had the same issue and I also used Esri and ArcGIS recently. However, it appears to happen in the computer that never used ArcGIS as well. I hope that someone can solve this.

**Comment by cm-usuda-keisuke** [comment]
I tried the following changes to ride.html that announced in #324 and solved it. I am not sure why, but you might try it.

> if you are using js.arcgis.com in your html - change the version from 4.3 to 4.6 and redeploy
> <script src="https://js.arcgis.com/4.6/"></script>

**Comment by robertjsandor** [comment]
The AWS folks mentioned that. 4.5 also works. I'd like to figure out someday how to load a different background. Thanks for the comment.

**Comment by mindacrobatic** [comment]
I just tried WildRydes an had the same issue.

**Comment by niravmsoni** [comment]
Just applied this change onto ride.html file
- change the version from 4.3 to 4.6 and redeploy
<script src="https://js.arcgis.com/4.6/"></script>
And it works!

**Comment by Delali97** [comment]
Experiencing the same issue, changed the version and it is still blank :(

**Comment by robertjsandor** [comment]
Yup, looks like js.argis.com deprecated 4.5 in the last few days. 4.6 now appears to be the working version.

**Comment by MattShirilla** [comment]
Just confirming what the others said.  I updated     `<link rel="stylesheet" href="https://js.arcgis.com/4.6/esri/css/main.css"> ` and `<script src="https://js.arcgis.com/4.6/"></script>` and things work perfectly.
